README.md

```diff
@@ -77,9 +77,9 @@ So what did we do here? Let's break it down step by step:
 
 1. First, we initialize our model and a `MessageGraph`.
 2. Next, we add a single node to the graph, called `"oracle"`, which simply calls the model with the given input.
-3. We add an edge from this `"oracle"` node to the special string `END`. This means that execution will end after current node.
+3. We add an edge from this `"oracle"` node to the special string `END` (`"__end__"`). This means that execution will end after the current node.
 4. We set `"oracle"` as the entrypoint to the graph.
-5. We compile the graph, ensuring that no more modifications to it can be made.
+5. We compile the graph, ensuring that it can be run.
```

Comment:
should we be more explicit here and say something like "we compile the graph, translating it to low-level pregel operations?"